src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -499,6 +515,10 @@ export default {
 			}
 		},
 
+		createConversationIfNotFound(name) {
+			this.$refs.newGroupConversation.createConversationIfNotFound(name)
```

Comment:
```suggestion
			this.$refs.newGroupConversation.createConversation(name)
```
Child component doesn't know, what the conversation is not found, so we shouldn't name the method like that

Reply:
There is already a function with that name for creating the conversation using the dialog, perhaps I should just put the function directly in leftSidebar without using the newGroupConversation component?

Reply:
You are right, that will be better in our case